### DISCUSSION TITLE
Github noob: Most efficient way to open projects?

### DISCUSSION THREAD

**Discussion by kylephelps757** [community/community#23024] [New to GitHub]
<p>Hi guys. I just started my Github account. I found out how to narrow the search down to the issues i want to tackle. What’s the most efficient way to open the projects with all of the various environments? Do I need to have all of the different software loaded on my PC or is there a better way?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>I’m not sure exactly what you mean by “open the projects with all of the various environments”. If you mean that you’re trying to configure your development environment to build a project hosted on GitHub from source, many open source projects have <a href="https://help.github.com/en/articles/creating-a-default-community-health-file-for-your-organization#about-default-community-health-files" rel="nofollow noopener">a CONTRIBUTING guide</a> that gives specific instructions on how to contribute to the project in multiple ways, including building from source.</p>
<p>I hope that helps! Let us know if you have more questions.</p>

**Comment by tsyoung1** [comment]
<p>Is that the simple way</p>
